src/scenes/HomePage/components/Works/components/Item/Item.module.scss

```diff
@@ -0,0 +1,27 @@
+@import '@styles/variables/all';
+@import '@styles/mixins/all';
+@import '@functions/functions';
+
+.item {
+  flex: 0 0 100%;
+  max-width: hl-grid-multiplier(31);
+  &.item {
+    margin-right: hl-grid-multiplier(1);
+  }
+  &:nth-child(2n) {
+    max-width: hl-grid-multiplier(20);
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    .image:first-child {
+      margin-bottom: hl-grid-multiplier(1);
+    }
+    .image:nth-child(n + 2) {
+      width: calc(50% - (1 / 2 * 100vw / 64));
```

Comment:
Could you explain this math? I don't understand this to be honest

Reply:
Space between last two image equal 1 column, for this reason i subtract half column from each image.

Reply:
can we do somehting like this? 
with easier calculations

Reply:
No, i use flex-wrap, and if i will do width 50% and margin, last image will go to next row, i cant use padding, i can just use width. 
I can do "width: calc (50% - var(--col-width)/2), this variant will be good ?

Reply:
yeah, let's do that